omnibridge/cli/argument_parser/argument_parser.py

```diff
@@ -1,34 +1,45 @@
-    return add_dalle_model_parser
-
-
-def add_list_wrappers_arguments(parser: Any) -> Any:
-    list_wrappers_parser = parser.add_parser(Commands.LIST_WRAPPERS.value, help="list wrappers.")
-    return list_wrappers_parser
+
+def add_create_model_sub_parser(parser: Any) -> None:
+    create_model_parser = parser.add_parser("model", help="Create a new model.")
+    create_model_parser.add_argument('model_type', choices=[m.value for m in SupportedModels],
+                                      help="The type of model to create.")
+    create_model_parser.add_argument('-n', '--name', type=str, required=True, help="name of the model.")
+    create_model_parser.add_argument('-k', '--key', type=str, required=True, help="api key name.")
+    create_model_parser.add_argument('--sub-model', type=str,
+                                      help="the sub model of the model youv'e chosen, for example" \
+                                        "'gpt-3.5-turbo' if chatgpt is chosen")
+    create_model_parser.add_argument('--num-images', type=str, default="4",
+                                      help="number of images per prompt, default 4.")
+    create_model_parser.add_argument('-r', '--res', type=str, default="256x256", help="resolution of images.")
+
+
+def add_create_flow_sub_parser(parser: Any) -> None:
+    create_flow_parser = parser.add_parser("flow", help="Create a new flow.")
+    create_flow_parser.add_argument('-n', '--name', type=str, required=True, help="name of the flow.")
+    create_flow_parser.add_argument('-t', '--type', type=str, choices=['seq', 'branching'], default='branching',
+                                          help="type of the flow.")
+    create_flow_parser.add_argument('-m', '--model', type=str,
+                                          help="name of the model to use.")
+    create_flow_parser.add_argument('-i', '--instruction', nargs='*', type=str,
+                                          help="instruct the model to do something with the input.")
+    create_flow_parser.add_argument('--multi', nargs='*', type=str,
+                                          help="use multiple models. Currently only supported in sequential flow.")
+
+
+def add_create_key_sub_parser(parser: Any) -> None:
+    create_key_parser = parser.add_parser("key", help="Create an api key.")
+    create_key_parser.add_argument('-n', '--name', help="name of the api key.", type=str, required=True)
+    create_key_parser.add_argument('-v', '--value', help="value of the api key.", type=str, required=True)
+
+
+def add_run_flow_sub_parser(parser: Any) -> None:
+    run_flow_parser = parser.add_parser("run-flow", help="run flow.")
+    run_flow_parser.add_argument('-n', '--name', type=str, required=True, help="name of the flow.")
+    run_flow_parser.add_argument('-p', '--prompt', help="prompt for flow", type=str, required=True)
+
+
+def add_list_wrappers_arguments(parser: Any) -> None:
+    parser.add_parser("list-wrappers", help="list wrappers.")
```

Comment:
overall, I think we should organize the files and directories based on their command, in the future where we'll have 10+ commands, it will become difficult understanding where to go

Reply:
split into 3 files for create, list, run